src/templates/contacts.html.j2

```diff
@@ -0,0 +1,64 @@
+{% extends "base.html.j2" %}
+{% block title -%}
+OSG Contacts
+{%- endblock %}
+{% block content %}
+{% if not authorized %}
+<div class="alert alert-info" role="alert">
+  To see contact information, load your grid certificate into your browser.
+</div>
+{% endif %}
+<div class="alert alert-info" role="alert">
+  Profile information is provided by the user and may be out of date.
+</div>
+<div class="container">
+  {%- for user in users %}
+    <div class="row text-white bg-dark">
```

Comment:
I think there needs to be some separation between the users.  If you don't want to add a style sheet, just add the style attribute to this div, so it would look like:

    <div class="row text-white bg-dark" style="margin-top:1em;">